logsrvd/logsrvd.c

```diff
@@ -342,6 +343,13 @@ handle_exit(ExitMessage *msg, struct connection_closure *closure)
 	closure->errstr = _("state machine error");
 	debug_return_bool(false);
     }
+    if (!closure->log_io) {
```

Comment:
this should not be a protocol error, so exit message can be stored for accept even if expect_iobufs=false.

Reply:
Fixed, thanks